src/webapp/components/dataset-wizard/FilterIndicators.tsx

```diff
@@ -135,7 +138,7 @@ export const FilterIndicators = React.memo((props: FilterIndicatorsProps) => {
 export type ChipFilterProps = {
     items: ChipItem[];
     label: string;
-    onChange: (item: ChipItem | ChipItem[]) => void;
+    onChange: (item: ChipItem[]) => void;
     value: string | string[];
```

Comment:
sorry, I hadn't seen that. Can we follow the same idea for value, to be string[]? 

The general idea is that, when using TS code in our control, we never have to call funcitons like isArray, isObject, isString, etc. (When we really need to have this kind of distinctions, in other scenarios, we use discriminating union types)